Demo cn0326

adrimbarean opened the pull request.

mirceacaprioru commented on projects/ADuCM3029_demo_cn0326/src/ADuCM3029_demo_cn0326.c:
```diff
@@ -0,0 +1,98 @@
+#include "Timer.h"
+
+#include "Communication.h"
+#define SPI0_SCLK_PORTP0_MUX  ((uint16_t) ((uint16_t) 1<<0))
```
Already defined in pinmux_config.c

stefpopa commented on projects/ADuCM3029_demo_cn0326/src/ADuCM3029_demo_cn0326.c:
```diff
@@ -0,0 +1,55 @@
+	/* Test communication with the board */
+	do {
+		test = AD7793_ReadRegister(AD7793_REG_ID);
+	} while((test & 0x0F) != 0x0B);
```
I would rather use defines and masks instead of "magic numbers" here

stefpopa commented on projects/ADuCM3029_demo_cn0326/src/CN0326.c:
```diff
@@ -0,0 +1,499 @@
+#if(USE_IOUT2 == YES)
+	f32current = iout2_calibration;
+#else
+	f32current = I_EXC ;
```
nitpick: extra white space before `;`

stefpopa commented on projects/ADuCM3029_demo_cn0326/src/CN0326.c:
```diff
@@ -0,0 +1,499 @@
+}
+
+
+
```
nitpick: 3 extra blank lines

stefpopa commented on projects/ADuCM3029_demo_cn0326/src/CN0326.c:
```diff
@@ -0,0 +1,499 @@
+	printf("Temperature = %.2f[˚C]\n", temp);
+}
+
+
```
nitpick: one new line should be enough

stefpopa commented on projects/ADuCM3029_demo_cn0326/src/CN0326.c:
```diff
@@ -0,0 +1,499 @@
+	}
+}
+
+
```
nitpick: extra new line

stefpopa commented on projects/ADuCM3029_demo_cn0326/src/CN0326.c:
```diff
@@ -0,0 +1,499 @@
+	*d = '\0';
+}
+
+
```
nitpick: extra new line

stefpopa commented on projects/ADuCM3029_demo_cn0326/src/CN0326.c:
```diff
@@ -0,0 +1,499 @@
+
+		} else {
+			uart_tbusy = UART_FALSE;
+		}
```
indentation in this function looks very strange

stefpopa commented on projects/ADuCM3029_demo_cn0326/src/CN0326.c:
```diff
@@ -0,0 +1,499 @@
+		timer_sleep(300);
+
+		printf("Calibration completed for %s channels!\n", arg);
+	} else {
```
I would use a` switch(case)` instead of several `else/else if` statements, as it makes the code more readable

adrimbarean commented:
changelog v1 -> v2:
* removed redeclaration of SPI0_SCLK_PORTP0_MUX
* used defines and masks instead of magic numbers
* corrected extra white spaces
* corrected extra white lines
* ran astyle

stefpopa commented:
as a general remark, add more info in the commit message

stefpopa commented:
as a general remark, add more info in the commit message; otherwise looks good!

adrimbarean commented:
changelog v2 -> v3:
* modify commit text

mirceacaprioru commented on projects/ADuCM3029_demo_cn0326/src/Communication.c:
```diff
@@ -0,0 +1,436 @@
+**/
+uint32_t SPI_Read(uint8_t ui8address, uint8_t ui8bytes)
+{
+
```
an extra line here

mirceacaprioru commented on projects/ADuCM3029_demo_cn0326/src/Communication.c:
```diff
@@ -0,0 +1,436 @@
+	spi_telegram.pReceiver 			= NULL; /* No received data */
+	spi_telegram.ReceiverBytes 		= 0;
+	spi_telegram.nRxIncrement 		= 0;
+	spi_telegram.bDMA 				= false; /* No DMA */
```
no aligned here

adrimbarean commented:
changelog v3 -> v4:
* enforce 80 column limit
* corrected if-else brackets {}
* removed extra line
* aligned assignation
